src/ripple/nodestore/impl/Shard.cpp

```diff
@@ -55,9 +55,8 @@ Shard::Shard(
 bool
 Shard::open(Scheduler& scheduler, nudb::context& ctx)
 {
+    std::lock_guard<std::mutex> lock(mutex_);
```

Comment:
We no longer need the `<std::mutex>` in this declaration.  Indeed, @seelabs is  just finishing up a PR that removes all of them.
```
std::lock_guard lock{mutex_};
```

Reply:
I was going to wait for its merge before opening up the flood gates.

Reply:
I wouldn't remove any `<std::mutex>`.  I just think the thing to do now is not add any more.